city_scrapers/spiders/chi_ssa_8.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+import re
+import unicodedata
+from calendar import day_name, month_name
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from w3lib.html import remove_tags
+
+
+class ChiSsa8Spider(CityScrapersSpider):
+    name = "chi_ssa_8"
+    agency = "Chicago Special Service Area #8 Lakeview East"
+    timezone = "America/Chicago"
+    start_urls = ["https://lakevieweast.com/ssa-8/"]
+
+    months = {m.lower() for m in month_name[1:]}
+    days = {d.lower() for d in day_name[1:]}
+
+    def parse(self, response):
+        for item in response.css(".post-content ul"):
+            bodyText = item.extract()
+            if "Location" in bodyText:
+                year = self._parse_year(item)
+                if year is not None:
+                    location = self._parse_location(item)
+                    for li in item.css("li"):
+                        startDate = self._parse_start(li, year)
+                        if startDate is not None:
+                            meeting = Meeting(
+                                title=self._parse_title_desc(li),
+                                description=self._parse_title_desc(li),
+                                classification=self._parse_classification(li),
+                                start=startDate,
+                                end=self._parse_end(li),
+                                all_day=self._parse_all_day(li),
+                                time_notes=self._parse_time_notes(li),
+                                location=location,
+                                links=self._parse_links(li),
+                                source=self._parse_source(response),
+                            )
+                            meeting["status"] = self._get_status(meeting)
+                            meeting["id"] = self._get_id(meeting)
+
+                            yield meeting
+
+    def _parse_year(self, item):
+        header = item.xpath("./preceding::h3[1]")
+        if header[0].extract():
+            headerString = header[0].extract()
+            if re.search("[1-3][0-9]{3}", headerString):
```

Comment:
We might be able to simplify this regex with `\d{4}`, or maybe just `2\d{3}` if we want to be more restrictive